video_search.py

```diff
@@ -0,0 +1,59 @@
+import json
+import pdb
```

Comment:
Try not to leave pdb around. In fact never leave any library imported that you are not using. Since you will not use pdb when this code is deployed on heroku, why leave it as well.